README.md

```diff
@@ -105,8 +106,9 @@ let dataset = new Dataset({
       resolve(data);
     });
     return this.store.query('record', params).then((data) => {
-      stats.totalPages = data.get('totalPages');
-      return data.get('records');
+      let meta = result.get('meta');
```

Comment:
There is a mistake here. You called the function parameter "data" but are referring to "result".